postgres/README.md

```diff
@@ -352,6 +401,22 @@ Then, [instrument your application container that makes requests to Postgres][32
 
 [Run the Agent's status subcommand][9] and look for `postgres` under the Checks section.
 
+## Deep Database Monitoring (beta)
```

Comment:
This is fine where it is for now, but I think this needs to work its way into the main agent conf section eventually.